pkg/transform/configmaps/configmaps.go

```diff
@@ -0,0 +1,59 @@
+package configmaps
+
+import (
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+// ConfigMap represent configmap definition
+type ConfigMap struct {
+	APIVersion string   `yaml:"apiVersion"`
+	Kind       string   `yaml:"kind"`
+	Metadata   MetaData `yaml:"metadata"`
+	Data       Data     `yaml:"data"`
+}
+
+// MetaData configmap's metadata
+type MetaData struct {
+	Name      string `yaml:"name"`
+	Namespace string `yaml:"namespace"`
+}
+
+// Data contains CA
+type Data struct {
+	CAData string `yaml:"ca"`
+}
+
+const (
+	// APIVersion is the apiVersion string
+	APIVersion = "v1"
+	// Namespace is the config map's default namespace string
+	Namespace = "openshift-config"
+	// Kind is config map resource type
+	Kind = "ConfigMap"
+)
+
+// GenConfigMap generates a secret
+func GenConfigMap(name string, CAData []byte) *ConfigMap {
```

Comment:
Maybe we need to move to a more generic function and pass the Namespace as parameters to the builder so we could create ConfigMaps for any namespace?

Reply:
Makes sense, I will change it